src/index.ts

```
@@ -18,14 +18,21 @@ const invalidRouteHandler = sharedMiddlewares.get('invalidRoute');
 app.set('port', port);
 app.set('logger', logger);
 app.disable('x-powered-by');
-// app.use(helmet());
+app.use(helmet({
```

ESLint

[error] Insert `⏎····`

(prettier/prettier)